Documentation/ceph-object.md

```diff
@@ -148,7 +148,7 @@ Now that the user connection variables were set above, we can proceed to perform
 Create a bucket in the object store
 
    ```bash
-   s3cmd mb --no-ssl --host=${AWS_HOST} --host-bucket=  s3://rookbucket
```

Comment:
Is the meaning of the preexisting line `--host-bucket=""` or `--host-bucket="s3://rookbucket`? Could this PR also make this more explicit since the line is already being modified? It seems very confusing to me.

Reply:
AFAIK the intention is `--host-bucket=""` to disable host bucket.